src/main/java/de/letsdev/maven/plugins/ios/ProjectBuilder.java

```diff
@@ -297,7 +308,7 @@ private static void generateXcFramework(List<String> frameworkPaths, String fram
         buildCommand.append(" -output ");
         buildCommand.append(frameworkTargetName);
 
-        Utils.executeShellScript("execute-xcodebuild.sh", buildCommand.toString(), null, targetWorkDirectory);
+        Utils.executeShellScript("execute-xcodebuild.sh", buildCommand.toString(), null, workDirectory);
```

Comment:
warum gab es diese Umstellung hier? ohne die brauchst du wahrscheinlich auch die Änderung in Zeile 282 nicht

Reply:
Die wird gebraucht, da er sonst das directory mit den reports nicht findet

Reply:
Das ist im projectdrirectory

Reply:
hmm. Landen die framework/ipa/zip Dateien denn aktuell am Ende noch im target/ Verzeichnis?

Reply:
Bin mir aber grade selbst nicht so sicher ob das was da gezippt wird richtig ist durch den anderen Pfad, wäre warscheinlich besser wenn dir nochmal anguckst ob das wirklich das richtige ist, bin mir nicht sicher was da überhaupt gezippt werden soll

Reply:
kannst du es bitte so zurückstellen wie es vorher war? Und dann hängst du vor den Pfad für die Reports die ProjectDirectory vorne an

Reply:
jap